app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -629,6 +635,10 @@ class EventDetailsFragment : Fragment() {
         startActivity(Intent.createChooser(emailIntent, "Chooser Title"))
     }
 
+    private fun redirectToWebView(event: Event) {
+        openUrl(requireContext(), "https://${getString(R.string.FRONTEND_HOST)}/e/${event.identifier}")
```

Comment:
It is opening custom tabs, not webview.

Reply:
Also, it should be `pending/order-identifier` after creating attendees otherwise there will be no pending orders.

Reply:
> It is opening custom tabs, not webview.
oh, then it is open custom tabs, just like Eventbrite, I thought WebView and Custom Tab is the same thing.

> Also, it should be pending/order-identifier after creating attendees otherwise there will be no pending orders.
It will not redirect to the order page yet as a user haven't specify how many tickets they would like. let me send the GIF or video

Reply:
So we are always redirecting to frontend event if the user does not want to pay with PayPal. Right?

Reply:
yes, if the event has enabled Paypal option

Reply:
https://youtu.be/XMcEQWljy7g

Reply:
IMO we should redirect to pending payment page otherwise there will be no use of attendee app for registering tickets.

Reply:
> yes, if the event has enabled Paypal option

And other payment options, like stripe and offline payment modes?

Reply:
If there isn't paypal payment then the app works normally with other payment method

Reply:
I also agree with your point. The implementation above was just following Eventbrite. The main problem of redirecting pending order page is to keep the user logged in. I'll see what I can do with it

Reply:
> The implementation above was just following Eventbrite

Please check message on gitter for it

Reply:
> I'll see what I can do with it

We can execute javascript in webview with the access token to authenticate the user. I have tried but not able to do it as I'm not familiar with javascript and cookie manager for the webview.

Reply:
Will try it if can be done :+1: